Fortnox.NET/Services/OrderService.cs

```diff
@@ -61,5 +61,36 @@ public static async Task<Order> CreateOrderAsync(FortnoxApiRequest request, Orde
 
             return (await FortnoxAPIClient.CallAsync(apiRequest)).Data;
         }
+
+        /// <summary>
```

Comment:
Its awesome that you've started adding documentation to the library! This is something that we've been wanting to get started on but didn't get to yet. 💯 👍